packages/react-devtools-core/src/standalone.js

```diff
@@ -200,6 +200,14 @@ var DevtoolsUI = {
     return DevtoolsUI;
   },
 
+  setBrowserName(name) {
+    config.browserName = name;
+    if (panel) {
+      panel.props.browserName = name;
```

Comment:
If we do [`reload()`](https://github.com/facebook/react-devtools/blob/master/packages/react-devtools-core/src/standalone.js#L46) it will lose current session of RN (Shown `Waiting for roots to load...`), maybe just don't update it? I think call the method before `startServer` is enough in this case.